aerosol_dynamics/particle.py

```diff
@@ -0,0 +1,107 @@
+import numpy as np
```

Comment:
The level of documentation is insufficient; however, given that I wrote the previous iteration of these calculations, I will be happy to step in and add documentation after the fact. Next time, we will be blocking merge requests for any documentation violations --- hopefully automatically by the linter ;)

Reply:
That's odd, I definitely added documentation here; this appears to be the old pull request! Did I miss something? 
I tried to find information on the charge_ratio, digging into your research letter, but couldn't make enough sense of it to add documentation.

Reply:
No problem if you need to redo, and don't worry about documentation --- don't let that slow you down. I am a pretty verbose person and happy to do more of the documentation :)

Reply:
My proposal: Even if this is the old pr, don't worry about it. You can do another pr later. Let's work to get this through first.